### PR TITLE
Cybereason: add new integration

### PR DESCRIPTION
**Pull request by squioc**
- Rename the old documentation as Malop activities
- Create the documentation for the new integration

### CODE REVIEW

File: docs/xdr/features/collect/integrations/endpoint/cybereason_malop_activities.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- all suspicious network connections
+- all suspicious executions
+
+{!_shared_content/operations_center/detection/generated/suggested_rules_0de050fb-3f56-4c7a-a9b6-76bf5298a617_do_not_edit_manually.md!}
+
+{!_shared_content/operations_center/integrations/generated/cybereason-malop-cef_do_not_edit_manually.md!}
+
+## Configure
+
+This setup guide shows how to forward all MalOp activities to SEKOIA.IO.
```
Comment (PierrickV) [inline docs/xdr/features/collect/integrations/endpoint/cybereason_malop_activities.md:22]:
```suggestion
This setup guide will guide you into forwarding all MalOp activities to SEKOIA.IO.
```

**Review by PierrickV — APPROVED** [review]

File: docs/xdr/features/collect/integrations/endpoint/cybereason_malop.md [hunk 2]
```diff
@@ -1,51 +1,33 @@
...
-$ cybereason-forwarders/scripts/run_forwarder.sh config/<my new configuration>.json
-```
-
-### Enjoy your events
+### Pull events
 
-Go to the [Events page](https://app.sekoia.io/operations/events) and wait for your incoming events!
+To start to pull events, you have to: 
+1. Go to the [playbook page](https://app.sekoia.io/operations/playbooks) and create a new playbook with the [Fetch new events from Cybereason](../../../automate/library/cybereason) module. 
+2. Set up the module configuration with your username and your password.
```
Comment (PierrickV) [inline docs/xdr/features/collect/integrations/endpoint/cybereason_malop.md:31]:
```suggestion
2. Set up the module configuration with your username and password.
```

File: docs/xdr/features/collect/integrations/endpoint/cybereason_malop.md [hunk 3]
```diff
@@ -1,51 +1,33 @@
...
-{!_shared_content/operations_center/integrations/generated/cybereason-malop-cef_do_not_edit_manually.md!}
+{!_shared_content/operations_center/integrations/generated/cybereason-malop-json_do_not_edit_manually.md!}
 
 ## Configure
 
 This setup guide shows how to forward all MalOp activities to SEKOIA.IO.
 
+### Prerequisites
+
+To forward events produced by Cybereason to SEKOIA.IO, you will need your username and your password.
```
[the remark this answers is not in the document]
Comment (squioc) [reply docs/xdr/features/collect/integrations/endpoint/cybereason_malop.md]: Applied

File: docs/xdr/features/collect/integrations/endpoint/cybereason_malop.md [hunk 4]
```diff
@@ -1,51 +1,33 @@
...
-
-```bash
-$ cybereason-forwarders/scripts/run_forwarder.sh config/<my new configuration>.json
-```
-
-### Enjoy your events
+### Pull events
 
-Go to the [Events page](https://app.sekoia.io/operations/events) and wait for your incoming events!
+To start to pull events, you have to:
```
[the remark this answers is not in the document]
Comment (squioc) [reply docs/xdr/features/collect/integrations/endpoint/cybereason_malop.md]: Applied

File: docs/xdr/features/collect/integrations/endpoint/cybereason_malop_activities.md (hunk 1 above)
Comment (squioc) [reply docs/xdr/features/collect/integrations/endpoint/cybereason_malop_activities.md]: Applied

File: docs/xdr/features/collect/integrations/endpoint/cybereason_malop.md (hunk 2 above)
Comment (squioc) [reply docs/xdr/features/collect/integrations/endpoint/cybereason_malop.md]: Applied

File: docs/xdr/features/collect/integrations/endpoint/cybereason_malop_activities.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+### Create your intake
+
+On SEKOIA.IO, go to the [Intakes page](https://app.sekoia.io/operations/intakes/new) and generate a new intake with the `Cybereason MalOp Activities` format.
+Keep aside the intake key.
+
+### Setup the Syslog collector
+
+Check the [Rsyslog Transport](../../../ingestion_methods/rsyslog/) documentation to install and set up the syslog collector.
```
[the remark this answers is not in the document]
Comment (squioc) [reply docs/xdr/features/collect/integrations/endpoint/cybereason_malop_activities.md]: Applied

**Pull request merged**